subprojects/packagefiles/google-benchmark/meson.build

```diff
@@ -2,6 +2,7 @@ project(
   'benchmark',
   'cpp',
   version: '1.8.4',
+  meson_version: '>=0.48.0',
```

Comment:
Argh, this should actually be 0.49.0.  The CI report missed this because it only matters on Windows.